packages/babel-generator/test/fixtures/disableASCIIEncode/options.json

```diff
@@ -0,0 +1,4 @@
+{
+  "disableASCIIEncode": true,
+  "minified": true
```

Comment:
Hm, without `minified: true` execution does not reach to `jsesc`, see `if (!this.format.minified && raw != null) {`